deploy/Chart/templates/database/pv.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-sts-0
```

Comment:
I think this name could be simplified to just `database`. Otherwise, why `pv-sts-0`?